wandb/integration/keras/callbacks/model_checkpoint.py

```diff
@@ -0,0 +1,77 @@
+import os
+import wandb
+import shutil
+
+from tensorflow.keras import callbacks
+
+from ..keras import patch_tf_keras
+
+
+patch_tf_keras()
+
+
+class WandbModelCheckpointCallback(callbacks.Callback):
```

Comment:
I was thinking we should try to subclass `tf.keras.callbacks.ModelCheckpoint` to take care of the checkpointing logic and use `wandb.Artifact` to log the model.

Reply:
Great idea!!!